React/Base/macOS/RCTUIKit.m

```diff
@@ -402,6 +402,28 @@ - (void)layoutSubviews
   [super layout];
 }
 
+- (NSArray<RCTUIView *> *)reactZIndexSortedSubviews // [macOS
+{
```

Comment:
Was this copied from the paper implementation by any chance?

Reply:
Yes. I pulled it in to have the same hitTest implementation as paper for views.

Reply:
OK! For any other comments I left where the answer is "It came from our paper implementation", feel free to comment and resolve!